go/tools/builders/nogo_validation.go

```diff
@@ -8,21 +8,53 @@ import (
 func nogoValidation(args []string) error {
 	validationOutput := args[0]
 	logFile := args[1]
+	nogoFixFileTmp := args[2]
+	nogoFixFile := args[3]
+
 	// Always create the output file and only fail if the log file is non-empty to
 	// avoid an "action failed to create outputs" error.
-	logContent, err := os.ReadFile(logFile);
+	logContent, err := os.ReadFile(logFile)
 	if err != nil {
 		return err
 	}
 	err = os.WriteFile(validationOutput, logContent, 0755)
 	if err != nil {
 		return err
 	}
+
+	nogoFixContent, err := os.ReadFile(nogoFixFileTmp)
+	if err != nil {
+		return err
+	}
+	err = os.WriteFile(nogoFixFile, nogoFixContent, 0755)
+	if err != nil {
+		return err
+	}
+
 	if len(logContent) > 0 {
```

Comment:
reduce nesting: https://github.com/uber-go/guide/blob/master/style.md#reduce-nesting

Reply:
It already followed the "reduce-nesting" and handled the error case first, right? 
+	nogoFixContent, err := os.ReadFile(nogoFixFileTmp)
+	if err != nil {
+		return err
+	}
+	err = os.WriteFile(nogoFixFile, nogoFixContent, 0755)
+	if err != nil {
+		return err
+	}